### PR TITLE
Removed stats, added daily telemetry

### PR DESCRIPTION
**Pull request by levb**
- Removed the old (unused?? cc @aaronrothschild) stats API
- Added daily telemetry for #instances, users, subscriptions

### CODE REVIEW

File: docs/administrator-guide/administrator-slash-commands.md [hunk 1]
```diff
@@ -23,6 +23,5 @@ Administrator slash commands are used to perform system-level functions that req
 * `/jira instance unalias [alias-name]` - Remove an alias from an instance
 * `/jira instance list` - List installed Jira instances
 * `/jira instance v2 <jiraURL>` - Set the Jira instance to process \"v2\" webhooks and subscriptions (not prefixed with the instance ID)
-* `/jira stats` - Display usage statistics
```
Comment (mickmister) [inline docs/administrator-guide/administrator-slash-commands.md:26]: Maybe add the `/jira setup` command, or do we not want to advertise that here?

File: server/plugin.go [hunk 2]
```diff
@@ -527,5 +507,51 @@ func (p *Plugin) trackWithArgs(name, userID string, args map[string]interface{})
...
+			cloud++
+		}
+	}
+	args["instance_count"] = server + cloud
+	if server > 0 {
+		args["server_instance_count"] = server
+	}
+	if cloud > 0 {
+		args["cloud_instance_count"] = cloud
+	}
```
Comment (mickmister) [inline server/plugin.go:536]: Do we want to instead unconditionally include these keys with count 0, in order to have telemetry payloads have a consistent structure? Not sure what the pros/cons are of _not_ doing that

**Review by mickmister — APPROVED** [review]
LGTM, just a few questions

**Review by aaronrothschild — APPROVED** [review]
LGTM.

File: server/plugin.go (hunk 2 above)
Comment (catalintomai) [reply server/plugin.go:536]: @sri-byte - not sure if this is a question or you or not - but do we have some (strong) preferences wrt to the 0 vs no value in telemetry payloads?

**Review by catalintomai — APPROVED** [review]

File: docs/administrator-guide/administrator-slash-commands.md (hunk 1 above)
Comment (levb) [reply docs/administrator-guide/administrator-slash-commands.md:26]: I thought we decided against that, wouldn't be in this PR anyway.

**Pull request merged**